index.html

```diff
@@ -583,6 +581,85 @@
       </section>
       <section>
         <h2>
+          Constrainable properties
+        </h2>
+        <p>
+          The <code>depthNear</code> and <code>depthFar</code> constrainable
+          properties are defined to apply only to <a>depth stream track</a>s.
+        </p>
+        <div class="note">
+          The <code>depthNear</code> and <code>depthFar</code> constrainable
+          properties, when set, allow the implementation to pick the best depth
+          camera mode optimized for the range <code>[depthNear,
+          depthFar]</code> and help minimize the error introduced by the lossy
+          conversion from the depth value <var>d</var> to a quantized
+          d<sub>8bit</sub> and back to an approximation of the depth value
+          <var>d</var>.
+        </div>
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>
+                Property name
+              </th>
+              <th>
+                Values
+              </th>
+              <th>
+                Notes
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <code>depthNear</code>
+              </td>
+              <td>
+                <code>ConstrainDouble</code>
+              </td>
+              <td>
+                The <a>near value</a>, in meters.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <code>depthFar</code>
+              </td>
+              <td>
+                <code>ConstrainDouble</code>
+              </td>
+              <td>
+                The <a>far value</a>, in meters.
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <p>
+          The <a>applyConstraints()</a> method MUST reject the promise with
+          <code>OverconstrainedError</code>, when invoked with
+          <code>depthNear</code> or <code>depthFar</code> property.
```

Comment:
I don't think that this is necessary.  Changing near and far for the purposes of changing where the 8-bits are applied seems reasonable.  You can maybe note that changing the operating mode of the camera might not be possible, which would result in an `OverConstrainedError` when the near and far constraints are outside the operating range.  However, as long as near and far are possible, I see no reason to prohibit this.  (Feel free to push this to a followup.)